src/env.ts

```diff
@@ -1,11 +1,29 @@
+export enum RenderBackend {
+  Core = 'core',
+  StorefrontRenderer = 'storefront-renderer',
+}
+
 export const env = {
   OAUTH2_DOMAIN: 'accounts.shopify.com',
   DEV_OAUTH2_DOMAIN: 'identity.myshopify.io',
   OAUTH2_CLIENT_ID: 'ff2a91a2-6854-449e-a37d-c03bcd181126',
   DEV_OAUTH2_CLIENT_ID: '1d7f695c-42e2-493a-a6dc-be12d4117d58',
-  OAUTH2_SUBJECT_ID:
-    '7ee65a63608843c577db8b23c4d7316ea0a01bd2f7594f8a9c06ea668c1b775c',
-  DEV_OAUTH2_SUBJECT_ID:
```

Comment:
Is this replaced by `DEV_OAUTH2_SUBJECT_NAME`?

Reply:
Yes. It now uses [dynamic registration](https://identity.docs.shopify.io/oauth/dynamic-registration-endpoint/) to the the client IDs. So that we don't have to hardcode UUID that depend on fixtures.

It was the approach recommended to me when I tried adding some seeds in the Identity Rails app.